specifications/api/swagger.yaml

```
@@ -4080,7 +4083,7 @@ components:
                 datum: *datum
                 assets: *walletAssets
         redeemers:
-            description: A list of redeemers data with their purpose.
+            description: A list of redeemers data with their purpose. The redeemers field in the `transaction` binary will be overwritten by this value.
```

Probably more
```suggestion
            description: A list of redeemers data with their purpose. The redeemers in the `transaction` binary will be overwritten by this value.
```
as I don't think we can say there's a field called redeemers in the binary.